src/__internal__/focus-trap/focus-trap.spec.tsx

```diff
@@ -86,37 +85,35 @@ const MockComponent = ({
   );
 };
 
+const defaultChildren = (
+  <>
+    <button type="button">{BUTTON_ONE}</button>
+    <button type="button">{BUTTON_TWO}</button>
+  </>
+);
+
+const renderMockComponent = (props: Partial<MockComponentProps> = {}) => {
+  const children = props.children || defaultChildren;
+  return render(<MockComponent {...props}>{children}</MockComponent>);
+};
+
 describe("FocusTrap", () => {
   describe("triggerRefocusFlag", () => {
     it("refocuses the last element that had focus within the trap when flag is set", () => {
       const { rerender } = render(
         <MockComponent autoFocus={false} triggerRefocusFlag={false}>
-          <button type="button">{BUTTON_ONE}</button>
-          <Select
-            id="c0499f86-d5a7-4a72-a0b7-753a2d218c54"
-            label="the dropdown"
-            aria-label="input"
-          >
-            <Option value="1" text="Option 1" />
-          </Select>
+          {defaultChildren}
```

Comment:
surely you don't need any children here if you're just passing in `defaultChildren`? You can just call `renderMockComponent` with the non-children props? Either that or not bother with the default inside the `renderMockComponent` definition.

(Same comment applies to other places where you've used `{defaultChildren"` as the children, of course. I won't comment on all of them separately!)

Reply:
I'm not sure this would work here as I need to call the `rerender` which is desctructured from the render function, I can do it like below but I'd still have to pass the same composed component back in so decided to leave it as is for these test cases.

```
      const { rerender } = renderMockComponent({autoFocus: false, triggerRefocusFlag: false});
  
      rerender(
        <MockComponent autoFocus={false} triggerRefocusFlag>
          {defaultChildren}
        </MockComponent>
      );
```

I've decided to do a slight refactor so that I can use re-use the function by just having it return the component and passing that into the `render`, so I can do something like below

```

 const { rerender } = render(renderMockComponent({autoFocus: false, triggerRefocusFlag: false}));
 
 rerender(renderMockComponent({autoFocus: false, triggerRefocusFlag: true}))
 ```